source/install/desktop-app-install.rst

```diff
@@ -18,138 +18,187 @@ Desktop application install guides
 
-Generic Linux package
-~~~~~~~~~~~~~~~~~~~~~
+    2. From the **\Downloads** folder, right-click on the file ``mattermost-desktop-setup-5.0.4-win.exe`` and select **Open** to start an installer for the app. Once finished, the Mattermost desktop app opens automatically.
 
-1. Download the latest version of the Mattermost desktop app:
+    **MSI Installer and Group Policies (Beta)**
 
-  - 64-bit systems: `mattermost-desktop-5.0.4-linux-x64.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x64.tar.gz>`__
-  - 32-bit systems: `mattermost-desktop-5.0.4-linux-ia32.tar.gz <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-ia32.tar.gz>`__
+    You can download the latest version of the Mattermost Desktop App MSI installer (Beta):
 
-2. Extract the archive to a convenient location. You can then execute ``mattermost-desktop``, which is located inside the extracted directory.
+    - MSI for `64-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x64.msi>`__
+    - MSI for `32-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x86.msi>`__
 
-3. To create a Desktop launcher, open the file **README.md** and follow the instructions in the **Desktop launcher** section.
+    See the `MSI installer and group policy documentation <https://docs.mattermost.com/install/desktop-msi-gpo.html>`__ for instructions on installing the Mattermost Desktop App via an MSI installer and configuring supported group policies. Changes to group policies require you to restart Mattermost for those changes to take effect.
 
-Ubuntu and Debian-based systems
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    The following group policies are available:
 
-Unofficial, community-driven ``.deb`` packages are available.
+    +----------------------------+-----------------------------------------------------------------------------+---------------------------------+
+    | Group policy               | Description                                                                 | Required Mattermost version     |
+    +============================+=============================================================================+=================================+
+    | Enable Server Management   | If disabled, management of servers in the app settings are disabled.        | 4.3 or later                    |
+    +----------------------------+-----------------------------------------------------------------------------+---------------------------------+
+    | Default Server List        | Define one or more default, permanent servers.                              | 4.3 or later                    |
+    +----------------------------+-----------------------------------------------------------------------------+---------------------------------+
```

Comment:
Missing the group policy for auto-updates

Reply:
Thanks, @esethna! @devinbinnie - Is this content you're able to help develop?

Reply:
Do we not already have documentation around changing Group Policy for the other settings? As per this https://github.com/mattermost/docs/pull/5628#discussion_r869607058 it's the same procedure.